### PR TITLE
C movie

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

**Review by matviichuk92 — COMMENTED** [review]
good job!

File: jv-cinema_shop.iml [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+      <configuration>
+        <datasource-map>
+          <unit-entry name="hibernate.cfg.xml" />
+        </datasource-map>
+        <naming-strategy-map />
+        <deploymentDescriptor name="hibernate.cfg.xml" url="file://$MODULE_DIR$/src/main/resources/hibernate.cfg.xml" />
+      </configuration>
+    </facet>
+  </component>
+</module>
```
Comment (KseniiaMakarova) [inline jv-cinema_shop.iml]: this file should not be here

I can see that in your master branch you have redundant files
they must be deleted  from master - see Sofiia's video tutorial

File: pom.xml [hunk 2]
```diff
@@ -25,6 +25,21 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
```
Comment (KseniiaMakarova) [inline pom.xml]: do we need junit dependency?

File: src/main/java/com/cinema/shop/util/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingException("Error creating Session Factory", e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/shop/util/HibernateUtil.java]:
```suggestion
            throw new RuntimeException("Error creating Session Factory", e);
```

this exception has a different nature

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (margarita-ignatieva) [reply pom.xml]: no, just copied pom from HW with it

File: jv-cinema_shop.iml (hunk 1 above)
Comment (margarita-ignatieva) [reply jv-cinema_shop.iml]: deleted from master but they are still here, probably after merge they'll be gone, or I need to pull changes from master to this branch?

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**